app/src/main/res/layout/item_message.xml

```diff
@@ -75,12 +75,47 @@
                     android:layout_height="wrap_content"
                     android:layout_marginStart="@dimen/marginStandardVerySmall"
                     android:src="@drawable/ic_certified"
-                    app:layout_constraintBottom_toBottomOf="@+id/expeditorName"
-                    app:layout_constraintEnd_toStartOf="@id/shortMessageDate"
-                    app:layout_constraintStart_toEndOf="@+id/expeditorName"
-                    app:layout_constraintTop_toTopOf="@+id/expeditorName"
+                    app:layout_constraintBottom_toBottomOf="@id/expeditorName"
+                    app:layout_constraintEnd_toStartOf="@id/iconsSpace"
+                    app:layout_constraintStart_toEndOf="@id/expeditorName"
+                    app:layout_constraintTop_toTopOf="@id/expeditorName"
                     tools:ignore="ContentDescription" />
 
+                <Space
+                    android:id="@+id/iconsSpace"
+                    android:layout_width="@dimen/marginStandardVerySmall"
+                    android:layout_height="0dp"
+                    app:layout_constraintBottom_toBottomOf="@id/iconCertified"
+                    app:layout_constraintEnd_toStartOf="@id/scheduleSendIcon"
+                    app:layout_constraintStart_toEndOf="@id/iconCertified"
+                    app:layout_constraintTop_toTopOf="@id/iconCertified" />
+
+                <ImageView
+                    android:id="@+id/scheduleSendIcon"
+                    android:layout_width="@dimen/smallIconSize"
+                    android:layout_height="@dimen/smallIconSize"
+                    android:layout_marginHorizontal="@dimen/marginStandardVerySmall"
+                    android:contentDescription="@string/contentDescriptionScheduleSendIcon"
+                    android:src="@drawable/ic_scheduled_messages"
+                    android:visibility="gone"
+                    app:layout_constraintBottom_toBottomOf="@id/shortMessageDate"
+                    app:layout_constraintEnd_toStartOf="@id/shortMessageDate"
+                    app:layout_constraintStart_toEndOf="@id/iconsSpace"
+                    app:layout_constraintTop_toTopOf="@id/shortMessageDate"
+                    app:tint="@color/scheduledIconColor"
+                    tools:visibility="visible" />
```

Comment:
The icon is not aligned vertically with the checkmark. I don't know if fixing it is complicated